### ISSUE TITLE
Enable fast-forward

### ISSUE THREAD

**Issue by edysegura** [agencyenterprise/notehacker#16]
When you join later in a meeting let user fast-forward the timer

**Comment by edysegura** [comment]
```js
var hms = '02:04:33';   // your input string
var a = hms.split(':'); // split it at the colons

// minutes are worth 60 seconds. Hours are worth 60 minutes.
var seconds = (+a[0]) * 60 * 60 + (+a[1]) * 60 + (+a[2]); 

console.log(seconds);
```

**Comment by edysegura** [comment]
This was closed here dcd1bcb7a9001660c4fa30d4d4a42c38809ba590
